src/db.c

```diff
@@ -586,20 +586,25 @@ void signalModifiedKey(client *c, redisDb *db, robj *key) {
 
 void signalFlushedDb(int dbid, int async) {
     int startdb, enddb;
+    robj *msgobj;
     if (dbid == -1) {
         startdb = 0;
         enddb = server.dbnum-1;
     } else {
         startdb = enddb = dbid;
     }
+    if (async)
+        msgobj = createStringObject("async",sizeof("async")-1);
```

Comment:
```suggestion
        msgobj = createStringObject("async",strlen("async"));
```

Reply:
why should we call strlen here in case this is a static string?

Reply:
Yes, `sizeof` would be faster, but I'm worried that one day this string will be extracted and this line will need to be modified at the same time.
Like following
```
char *event = "async";
msgobj = createStringObject("async",sizeof(event)-1);
```

Perhaps we can use 5 directly, and avoid using sizeof or strlen.
```
msgobj = createStringObject("async",5);
```

Reply:
we can I just think that maintaining hardcoded numbers usually cause errors (when we will decide to change the string and then forget to match the size)

Reply:
here is a demo
```c
#include <string.h>
int main(void) {
    return strlen("abcde");
}
```
```s
main:
.LFB5:
        .cfi_startproc
        pushq   %rbp
        .cfi_def_cfa_offset 16
        .cfi_offset 6, -16
        movq    %rsp, %rbp
        .cfi_def_cfa_register 6
        movl    $5, %eax
        popq    %rbp
        .cfi_def_cfa 7, 8
        ret
        .cfi_endproc
```
From the assembly file, the function returns 5 directly without calling strlen